pkg/util/sys/device.go

```diff
@@ -206,11 +208,12 @@ func GetDeviceProperties(device string, executor exec.Executor) (map[string]stri
 // GetDevicePropertiesFromPath gets a device property from a path
 func GetDevicePropertiesFromPath(devicePath string, executor exec.Executor) (map[string]string, error) {
 	output, err := executor.ExecuteCommandWithOutput("lsblk", devicePath,
-		"--bytes", "--nodeps", "--pairs", "--paths", "--output", "SIZE,ROTA,RO,TYPE,PKNAME,NAME,KNAME")
+		"--bytes", "--nodeps", "--pairs", "--paths", "--output", "SIZE,ROTA,RO,TYPE,PKNAME,NAME,KNAME,MOUNTPOINT,FSTYPE")
 	if err != nil {
 		logger.Errorf("failed to execute lsblk. output: %s", output)
 		return nil, err
 	}
+	logger.Debugf("lsblk output: %q", output)
```

Comment:
lsblk and udevadm isn't terribly verbose, right? The osd prepare job just has debug logging enabled by default.

Reply:
No it's actually not bad but very useful for the prepare job when running :)